wasm.md

```diff
@@ -526,20 +533,29 @@ The `*_local` instructions are defined here.
 ### Globals
 
 ```k
-    syntax Instr ::= "init_global" Int Val
-                   | "(" "global.get" Int ")"
+    syntax Instr ::= "(" "global.get" Int ")"
                    | "(" "global.set" Int ")"
+                   | "(" "global.set" Int Instr ")"
  // -----------------------------------------
```

Comment:
Line up all the `")"` and make the `-----` extend to the end of the line:

```
    syntax Instr ::= "(" "global.get" Int       ")"
                   | "(" "global.set" Int       ")"
                   | "(" "global.set" Int Instr ")"
 // -----------------------------------------------
```

Reply:
fixed